### PR TITLE
plan controller (js) test

### PR DESCRIPTION
**Pull request by sidk**
Test for plan_controller.js

Also formats the plan_form erb file.

### CODE REVIEW

File: app/javascript/src/controllers/plan_controller.js [hunk 1]
```diff
@@ -1,4 +1,5 @@
 import { Controller } from "stimulus"
+import $ from "jquery"
```
Comment (savannidgerinel) [inline app/javascript/src/controllers/plan_controller.js]: Why is this necessary?

Comment (sidk) [reply app/javascript/src/controllers/plan_controller.js]: jest complains that `$` is undefined in the controller since its unaware of the webpacker environment.
